plans/basic.fmf

```diff
@@ -7,4 +7,4 @@ discover:
     how: fmf
     filter: 'tier: 1, 2'
 execute:
-    how: beakerlib.tmt
+    how: tmt
```

Comment:
In order to make tests working with cruncher, this still needs to use the old syntax. But @thrix mentioned that the full `tmt` support in packit should be coming soon. I try to get rid of this in #467 while keeping a special hack-plan to make the tests still working with cruncher.

Reply:
Ok, so I'll wait on #467 and see what is left to fix.

Reply:
The plan changes in #467 are ready for review. Please, have a look. Thanks.

Reply:
Rebased on top of current master (which has #467 merged)